sw/tools/generators/gen_radio.ml

```diff
@@ -39,7 +39,7 @@ let check_function_name = fun s ->
 
 
 let norm1_ppm = fun c ->
```

Comment:
the function `norm1_ppm2` should be updated the same way

Reply:
Oups, yes, done.